Dockerfile

```diff
@@ -3,6 +3,7 @@ LABEL contributor="shadowapex@gmail.com"
 COPY rootfs/etc/pacman.conf /etc/pacman.conf
 RUN echo -e "keyserver-options auto-key-retrieve" >> /etc/pacman.d/gnupg/gpg.conf && \
     pacman-key --init && \
+    pacman-key --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 1EB2638FF56C0C53 && \
```

Comment:
Van this be avoided if `auto-key-retrieve` is in effect with the change below?

Reply:
I just tested it and I think so. I'll push the change

Reply:
Great